### ISSUE TITLE
Add operator to support dynamic neural networks

### ISSUE THREAD

**Issue by EnfangCui** [kubeedge/sedna#40]
**Dynamic neural network** is an emerging research topic in deep learning. Compared to static models which have fixed computational graphs and parameters at the inference stage, dynamic networks can adapt their structures or parameters to different inputs, leading to notable advantages in terms of accuracy, computational efficiency, adaptiveness, etc.

**Comment by ugvddm** [comment]
Hi CEF， thanks for your contribution,
it is seem like  a  good idea, but we need find some scenaro  to evaluate it more better.

**Comment by EnfangCui** [comment]
> Hi CEF， thanks for your contribution,
> it is seem like a good idea, but we need find some scenaro to evaluate it more better.

Yes, we need to find some real-world application scenarios. I will continue to follow up on the progress of this technology, and when I have a valuable scenario, I will make a detailed proposal.

**Comment by ugvddm** [comment]
we look for it together :)
